### ISSUE TITLE
Make more mobile browser friendly

### ISSUE THREAD

**Issue by juhp** [alexwl/haskell-code-explorer#26]
I just tried it for the first, but happened to be on my phone - the layout (scrolling) is not so mobile browser friendly.

I am sure desktop browsing is the main target, maybe some smaller layout tweaks could improve the mobile experience. :)

**Comment by seagreen** [comment]
On my phone the main issue is that the file list takes up almost all the space by default. Once I hit "Hide" to the right of the file search box things are greatly improved, but I have to do that each time I go to haskell-code-explorer.

**Comment by alexwl** [comment]
Of course, the left panel should be hidden by default on small screens.

I've committed the fix (db54d0594b84966ec3568c90c4219abb60c2e41f) and updated [haskell-code-explorer.mfix.io](https://haskell-code-explorer.mfix.io). I've also reduced `initial-scale` from 1 to 0.6 to fit more text on the screen.

Any suggestions on how to make the UI more mobile-friendly are welcome.

**Comment by seagreen** [comment]
Fixed on my phone!
